### PR TITLE
DTGB-424: Fixed detail layout.

### PR DESCRIPTION
**Pull request by meirege**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: templates/core/fields/field--node--field-tags.html.twig [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+ * - field_type: The type of the field.
+ * - label_display: The display settings for the label.
+ *
+ *
+ * @see template_preprocess_field()
+ */
+#}
+
+{% if label_hidden %}
+  {%- for item in items %}
```
Comment (delrueba) [inline templates/core/fields/field--node--field-tags.html.twig]: If it's a 'for', shouldn't it be a list?
If so: check with items|length to provide a single item alternative.

**Review by delrueba — CHANGES_REQUESTED** [review]

File: templates/core/fields/field--node--field-tags.html.twig (hunk 1 above)
Comment (meirege) [reply templates/core/fields/field--node--field-tags.html.twig]: Changed it.

File: templates/core/fields/field--node--field-related--infopage.html.twig [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+%}
+{%
+  set title_classes = [
+    'field__label',
+    label_display == 'visually_hidden' ? 'visually-hidden',
+  ]
+%}
+
+{% if label_hidden %}
+  <ul{{ attributes.addClass(classes, 'field__items') }}>
```
Comment (delrueba) [inline templates/core/fields/field--node--field-related--infopage.html.twig:53]: Need a single item variant, we don't do single item lists.
(easier to theme too)

**Review by delrueba — CHANGES_REQUESTED** [review]

File: templates/core/fields/field--node--field-related--infopage.html.twig (hunk 2 above)
Comment (meirege) [reply templates/core/fields/field--node--field-related--infopage.html.twig:53]: Will refactor.

**Review by delrueba — APPROVED** [review]

**Pull request merged**